CIP-0060/cddl/version-1.cddl

```diff
@@ -97,6 +101,11 @@ metadata_details =
     ? parental_advisory: string,
     ? explicit: bool,
     ? isrc: string,
+    ? isrc: string,
```

Comment:
I'm guessing this line is duplicated because `isrc:` was already there before the whole group was recently added?

Reply:
Nope. this is wrong. I'll fix...

Reply:
fixed

Reply:
OK, I see what happened 👍